Subject: make sure dbus is built
From: Michael Olbrich <m.olbrich@pengutronix.de>

By default all tools are disabled when cross-compiling. For some reason qdbus
(libs and tools) is a "tool". This patch enables qdbus for cross-compiling.
Other tools like designer and assistant have libs that can be used by other
applications. These are enabled as well.
Note: "-make tools" is needed as a configure flag as well to enable this.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 projects.pro |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: b/projects.pro
===================================================================
--- a/projects.pro
+++ b/projects.pro
@@ -27,7 +27,7 @@ isEmpty(QT_BUILD_PARTS) { #defaults
 for(PROJECT, $$list($$lower($$unique(QT_BUILD_PARTS)))) {
     isEqual(PROJECT, tools) {
        !cross_compile:SUBDIRS += tools
-       else:SUBDIRS += tools/qtestlib
+       else:SUBDIRS += tools/qtestlib tools/qdbus tools/designer/src/lib tools/designer/src/uitools tools/assistant
     } else:isEqual(PROJECT, examples) {
        SUBDIRS += examples
     } else:isEqual(PROJECT, demos) {
